lib/queue_bus/subscription_list.rb

```diff
@@ -46,6 +46,12 @@ def add(sub)
       @subscriptions[sub.key] = sub
     end
 
+    def remove(sub)
```

Comment:
I'm having trouble understanding how this is being used with these changes.

Reply:
Ah - when I was first trying to understand how the repo worked, I thought the application would be adding / removing subscriptions based on this list, so I built this out first to get started with something.

Turned out that's not the case and the application just makes redis calls, and I followed suit there, meaning this part is no longer needed. Removed.